src/components/ChattyLLM/Message.vue

```diff
@@ -90,7 +90,7 @@ export default {
 	data: () => {
 		return {
 			displayName: getCurrentUser()?.displayName ?? getCurrentUser()?.uid ?? t('assistant', 'You'),
-			userId: getCurrentUser()?.uid ?? t('assistant', 'yooniquely-you'),
+			userId: getCurrentUser()?.uid ?? t('assistant', 'You'),
```

Comment:
If there is no user in the context, something is wrong, it does not matter that much to have a fallback there, right? Especially since this userId is only passed to the avatar. Anyway, can't hurt :+1: